### DISCUSSION TITLE
Disabling Navigation Bar

### DISCUSSION THREAD

**Discussion by vedxp** [vuejs/vitepress#1583] [Q&A]
Is there any way to disable Nav?

**Comment by brc-dd** [comment]
```
---
layout: false
---

no theme, no sidebar, no layout, no footer...
```

**Reply by vedxp** [reply]
> ```
> ---
> layout: false
> ---
> 
> no theme, no sidebar, no layout, no footer...
> ```

Isn't it possible to disable specifically the Nav only, either globally or individually by frontmatter?
